Rakefile

```
@@ -99,6 +99,11 @@ task :clean do
   end
 end
 
+desc "Build #{Bundler::GemHelper.gemspec.full_name} and move it to local ruby/ruby project's bundled gems folder"
+task build_for_local_ruby: :build do
+  sh "mv #{path}.gem ../ruby/gems/"
```

This has the implicit dependency of `ruby` being cloned on the same level as RDoc, right?

If so, how about we do this? Also, let's use `FileUtils` rather than `sh`, so that it's cross-platform.

```suggestion
  target = File.join("..", "ruby", "gems")

  unless File.directory?(target)
    abort("Expected ruby to be cloned at the same level as #{Bundler::GemHelper.gemspec.full_name} to use this task")
  end

  FileUtils.mv("#{path}.gem", target)
```

---

We can use `mv` in `Rakefile`. It's better than `FileUtils.mv` because `mv` outputs log: `mv("#{path}.gem", target)`

---

Will that work for Windows contributors though? Can we not just log manually and still ensure cross-platform support?

---

The `mv` here is an extension of `FileUtils`'s ([source](https://github.com/ruby/rake/blob/09a1cd41d13d0c2c23a0775f860a8c5e71d43911/lib/rake/file_utils_ext.rb#L33-L40)). So it should work cross-platform.